### ISSUE TITLE
Energy groups makes gmx 2018 fall back to CPU

### ISSUE THREAD

**Issue by jbarnoud** [Tsjerk/gromit#22]
In gmx 5 and 2016, energy groups where ignored when running on GPU. In gmx 2018, energy groups makes mdrun fall back to CPU which leads to worst performances. Energy groups should not be set by default by gromit and martinate.

**Comment by jmelcr** [comment]
I totally agree. Workstations often have accelerators nowadays and this prevents their utilization. After all, why would we like to track energies of groups during the preparation stages?
